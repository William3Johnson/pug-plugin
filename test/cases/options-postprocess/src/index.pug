html
  head
    link(href=require('./styles.scss') rel='stylesheet')
    script(src=require('./main.js') defer)
  body
    h1 Hello Pug!
    p REPLACE ME