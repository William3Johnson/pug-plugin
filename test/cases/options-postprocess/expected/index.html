<!DOCTYPE html><html><head><link href="/assets/css/styles.81f40c66.css" rel="stylesheet"><script src="/assets/js/main.53b9cc33.js" defer></script></head><body><h1>Hello Pug!</h1><p>The new replaced content.</p></body></html>